rxbindings/src/main/java/com/amplifyframework/rx/RxApi.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.rx;
+
+import com.amplifyframework.api.ApiCategoryBehavior;
+
+/**
+ * An Rx-idiomatic expression of Amplify's {@link ApiCategoryBehavior}s.
+ */
+@SuppressWarnings("WeakerAccess") // This is a public API.
+public interface RxApi extends RxRestApi, RxGraphQlApi {
+}
```

Comment:
```java
public interface RxApiCategoryBehavior extends RxRestBehavior, RxGraphQLBehavior {}
```

Also would be nice to agree on `GraphQL` vs `GraphQl` when naming our java classes. 

On somewhat-related note, there is also similar inconsistency in datastore storage engine (`Sql` vs `SQL`, `SQLite` vs `Sqlite`)

Reply:
there is this guidance - 

https://stackoverflow.com/a/15526526/695787

But it’s complicated since the acronyms are embedded in other words. I guess it should be `GraphQL` and `Sqlite` in pascal-case, and `graphql` and `sqlite` in camel-case? Does that sound right?